common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -121,6 +150,17 @@
                 }
             };
 
+            NewPostView.prototype.togglePostVisibility = function($target, force_cohorted) {
```

Comment:
Can we name this something other than `togglePostVisibility` ? That sounds to me like this method will mutate the state and invert the "visibility" value, which doesn't seem to be what it does.

maybe `updateVisibilityMessage` or similar?